### ISSUE TITLE
dropShadow won't be applied in power bi service

### ISSUE THREAD

**Issue by ibarrau** [deldersveld/PowerBI-ThemeTemplates#31]
Hi. I have tried this json and a custom one over different reports and workspaces. The thing is that the shadow of the visualizations only change in power bi desktop. After you publish them to Service, the reports won't have any shadow at all. Do you know something about it?

**Comment by deldersveld** [comment]
No, but if the setting still works in Desktop, my guess is it's going to be an issue with Service and not the JSON.
